api/v1alpha3/awscluster_webhook.go

```diff
@@ -80,10 +85,38 @@ func (r *AWSCluster) ValidateUpdate(old runtime.Object) error {
 		)
 	}
 
+	allErrs = append(allErrs, r.validateCIDRBlocks()...)
+
 	return aggregateObjErrors(r.GroupVersionKind().GroupKind(), r.Name, allErrs)
 }
 
+func (r *AWSCluster) validateCIDRBlocks() []*field.Error {
+	var errs field.ErrorList
+
+	if r.Spec.Bastion.DisableIngressRules && len(r.Spec.Bastion.CIDRBlocks) > 0 {
+		errs = append(errs,
+			field.Forbidden(field.NewPath("spec", "Bastion", "CIDRBlocks"), "cannot be set if spec.Bastion.DisableIngressRules is true"),
```

Comment:
Field path segments need to be lowercase

Reply:
thanks for catching @ncdc ... resolved.